lib/shoulda/matchers/rails_shim.rb

```diff
@@ -58,12 +46,7 @@ def generate_validation_message(
         end
 
         def make_controller_request(context, verb, action, request_params)
```

Comment:
Now that there is no logic in `make_controller_request`, `tables_and_views`, and `validation_message_key_for_association_required_option`, what do you think about folding this code into the place where they are used so that we can remove these methods?

Reply:
I think this is a good idea! I'll do that asap. Thanks!

Reply:
Done!